s3-sink-connector/src/main/java/io/aiven/kafka/connect/s3/config/S3SinkConfig.java

```diff
@@ -181,76 +112,42 @@ private static Map<String, String> handleDeprecatedYyyyUppercase(final Map<Strin
-                AWS_SECRET_ACCESS_KEY_CONFIG);
-
-        configDef.define(AWS_CREDENTIALS_PROVIDER_CONFIG, Type.CLASS, AWS_CREDENTIAL_PROVIDER_DEFAULT,
-                Importance.MEDIUM,
-                "When you initialize a new " + "service client without supplying any arguments, "
-                        + "the AWS SDK for Java attempts to find temporary "
-                        + "credentials by using the default credential " + "provider chain implemented by the "
-                        + "DefaultAWSCredentialsProviderChain class.",
-
-                GROUP_AWS, awsGroupCounter++, ConfigDef.Width.NONE, AWS_CREDENTIALS_PROVIDER_CONFIG);
-
-        configDef.define(AWS_S3_BUCKET_NAME_CONFIG, Type.STRING, null, new BucketNameValidator(), Importance.MEDIUM,
-                "AWS S3 Bucket name", GROUP_AWS, awsGroupCounter++, ConfigDef.Width.NONE, AWS_S3_BUCKET_NAME_CONFIG);
-
-        // AWS S3 Server Side Encryption Algorithm configuration
-        // Example values: 'AES256' for S3-managed keys, 'aws:kms' for AWS KMS-managed keys
-        configDef.define(AWS_S3_SSE_ALGORITHM_CONFIG, Type.STRING, null, new ConfigDef.NonEmptyString(),
-                Importance.MEDIUM, "AWS S3 Server Side Encryption Algorithm. Example values: 'AES256', 'aws:kms'.",
-                GROUP_AWS, awsGroupCounter++, ConfigDef.Width.NONE, AWS_S3_SSE_ALGORITHM_CONFIG);
-
-        configDef.define(AWS_S3_ENDPOINT_CONFIG, Type.STRING, null, new UrlValidator(), Importance.LOW,
-                "Explicit AWS S3 Endpoint Address, mainly for testing", GROUP_AWS, awsGroupCounter++,
-                ConfigDef.Width.NONE, AWS_S3_ENDPOINT_CONFIG);
-
-        configDef.define(AWS_S3_REGION_CONFIG, Type.STRING, null, new AwsRegionValidator(), Importance.MEDIUM,
-                "AWS S3 Region, e.g. us-east-1", GROUP_AWS, awsGroupCounter++, ConfigDef.Width.NONE,
-                AWS_S3_REGION_CONFIG);
-
-        configDef.define(AWS_S3_PART_SIZE, Type.INT, S3OutputStream.DEFAULT_PART_SIZE, new ConfigDef.Validator() {
-
-            static final int MAX_BUFFER_SIZE = 2_000_000_000;
-
-            @Override
-            public void ensureValid(final String name, final Object value) {
-                if (value == null) {
-                    throw new ConfigException(name, null, "Part size must be non-null");
-                }
-                final var number = (Number) value;
-                if (number.longValue() <= 0) {
-                    throw new ConfigException(name, value, "Part size must be greater than 0");
-                }
-                if (number.longValue() > MAX_BUFFER_SIZE) {
-                    throw new ConfigException(name, value,
-                            "Part size must be no more: " + MAX_BUFFER_SIZE + " bytes (2GB)");
-                }
-            }
-        }, Importance.MEDIUM,
+    private static void addS3SinkConfig(final ConfigDef configDef) {
```

Comment:
this method name looks too generic

Reply:
So this is specific S3Sink only configuration, I just followed the naming convention of our other config statics, I am open to an alternative if you have one though.

Reply:
For now, its only adding part size config. So may be addS3PartSizeConfig is ok.

Reply:
Sure we can always change it again if it expands in the future